### PR TITLE
Drop hashicorp multierror due to MPL

### PR DESCRIPTION
**Pull request by monopole**
https://github.com/cncf/foundation/blob/master/allowed-third-party-license-policy.md#cncf-allowlist-license-policy

ALLOW_MODULE_SPAN

### CODE REVIEW

File: api/filters/patchstrategicmerge/patchstrategicmerge_test.go [hunk 1]
```diff
@@ -692,7 +692,7 @@ spec:
     spec:
       containers:
       - name: consul
-        image: "hashicorp/consul:1.9.1"
+        image: "dashicorp/consul:1.9.1"
```
Comment (monopole) [inline api/filters/patchstrategicmerge/patchstrategicmerge_test.go:695]: this is just so it doesn't show up in grep.

**Review by natasha41575 — APPROVED** [review]

**Comment by natasha41575** [comment]
@monopole it is failing the multi module check

**Comment by monopole** [comment]
sigh
/test  kustomize-presubmit-master

**Pull request merged**